lib/qtermwidget.json

```diff
@@ -1,6 +1,6 @@
 {
    "Name" : "QTermWidget",
-   "Version" : "2.1.0",
+   "Version" : "2.2.0",
    "Vendor" : "LXQt",
    "Copyright" : "(C) 2022-2024 LXQt",
```

Comment:
Might it be prudent to update `Copyright` year here perhaps?

Reply:
Thanks! Done.